crates/kubelet/src/handle.rs

```diff
@@ -190,3 +190,15 @@ impl<R: AsyncRead + AsyncSeek + Unpin, S: Stop> PodHandle<R, S> {
         Ok(())
     }
 }
+
+/// Generates a unique human readable key for storing a handle to a pod in a
+/// hash. This is a convenience wrapper around [crate::pod_key].
+pub fn key_from_pod(pod: &Pod) -> String {
+    pod_key(pod.namespace(), pod.name())
+}
+
+// Generates a unique human readable key for storing a handle to a pod if you
+// already have the namespace and pod name.
+pub fn pod_key<N: AsRef<str>, T: AsRef<str>>(namespace: N, pod_name: T) -> String {
```

Comment:
Does this have to be public? Seems like `key_from_pod`  is enough

Reply:
yes, there are places we don't have a pod but do have a name

Reply:
Yeah, this is used for `logs` as it passes the namespace and name, but not the pod object